dashboard/test/ui/features/acquisition_products/pd/teacher_application.feature

```diff
@@ -1,46 +1,14 @@
 @dashboard_db_access
-@eyes
 
 Feature: Teacher Application
 
-Scenario: Teacher starts a new application and submits it
+@eyes
+Scenario: Teacher starts a new csd application and submits it with principal approval
```

Comment:
nit (and sorry if this is wrong): should this be "administrator approval"?

Reply:
It is principal for now but will be updated when the switch to administrator fully happens!